sklearn/linear_model/_coordinate_descent.py

```diff
@@ -2078,6 +2083,7 @@ def _is_multitask(self):
 
     def __sklearn_tags__(self):
         tags = super().__sklearn_tags__()
+        tags.input_tags.sparse = True
         tags.target_tags.multi_output = False
```

Comment:
I'm even wondering if the multi_output could also be moved together with teh `_is_multitask` method but it needs to be checked.

Reply:
done in 3ac38e1